service-node/backend/src/main/scala/org/enterprisedlt/fabric/service/node/process/DockerManagedBox.scala

```diff
@@ -409,6 +416,65 @@ class DockerManagedBox(
         }
     }
 
+
+    private def pullImageIfNeeded(image: String, forcePull: Boolean = false): Either[String, Unit] = {
+        if (forcePull) {
```

Comment:
this could be just:
`if(!forcePull && findImage(...).isDefined) { ... } else { pullImage(...) }`

Reply:
Done. Pls take a look.